docs/en/stack/ml/df-analytics/ml-dfa-regression.asciidoc

```diff
@@ -592,10 +850,27 @@ POST _ml/data_frame/_evaluate
 ====
 
 When you have trained a satisfactory model, you can deploy it to make predictions
-about new data. Those steps are not covered in this example. See
-<<ml-inference>>. 
+about new data.
 
 If you don't want to keep the {dfanalytics-job}, you can delete it. For example,
 use {kib} or the {ref}/delete-dfanalytics.html[delete {dfanalytics-job} API].
 When you delete {dfanalytics-jobs} in {kib}, you have the option to also remove
 the destination indices and index patterns.
+
+[discrete]
+[[ml-inference-reg]]
+== {infer-cap}
```

Comment:
I wonder if this inference section should precede the example? Otherwise it's harder to tell where the example ends. Is it a good pattern to always end this type of page with an example (where possible)?